src/Ng/Program.cs

```diff
@@ -59,17 +66,10 @@ public static void Main(string[] args)
                 _logger = loggerFactory.CreateLogger<Program>();
 
                 var cancellationTokenSource = new CancellationTokenSource();
-                if (args.Length == 0)
-                {
-                    throw new ArgumentException("Missing tool specification");
-                }
 
                 // Create an ITelemetryService
                 var telemetryService = new TelemetryService(new TelemetryClient());
 
-                var jobName = args[0];
-                LogContext.PushProperty("JobName", jobName);
```

Comment:
>LogContext [](start = 16, length = 10)

Why was this removed? Does it not work?

Reply:
Yes, it works, but only on traces. I added a more general solution for all AI events.